user_data/strategies/bbrsi.py

```diff
@@ -0,0 +1,128 @@
+# --- Do not remove these libs ---
+
+from freqtrade.persistence import Trade
+from freqtrade.strategy import IStrategy
+from typing import Dict, List
+from functools import reduce
+from pandas import DataFrame
+# --------------------------------
+
+import talib.abstract as ta
+import freqtrade.vendor.qtpylib.indicators as qtpylib
+import numpy  # noqa
+from wao.waostrategy import WAOStrategy
+
+
+class bbrsi(IStrategy):
+    # Minimal ROI designed for the strategy.
+    # This attribute will be overridden if the config file contains "minimal_roi"
+    waostrategy = WAOStrategy("bbrsi")
```

Comment:
this is wrong, bbrsi should extend waostrategy
and waostrategy should extend IStrategy, revert this and implement correctly

we dont want a controller, we want a parent class. I have make that very clear. 

all 5 strategies need to extend waostrategy instead of iStrategy

Reply:
OK but we need the strategy controller in waostrategy and when initializing it uses the "freq_strategyname" since this name is different for all the strategies how should I handle this?